---
title: "永远不会达到的值，因为 catch 块&lt;name1&gt;继承&lt;name2&gt;&quot; |Microsoft 文档"
ms.date: 2015-07-20
ms.prod: .net
ms.technology:
- devlang-visual-basic
ms.topic: article
f1_keywords:
- BC42029
- vbc42029
helpviewer_keywords:
- BC42029
ms.assetid: 4bc84a84-ec7e-49db-8794-4fb71d36d5c0
caps.latest.revision: 12
author: dotnet-bot
ms.author: dotnetcontent
translation.priority.ht:
- de-de
- es-es
- fr-fr
- it-it
- ja-jp
- ko-kr
- ru-ru
- zh-cn
- zh-tw
translation.priority.mt:
- cs-cz
- pl-pl
- pt-br
- tr-tr
ms.translationtype: Machine Translation
ms.sourcegitcommit: 9f5b8ebb69c9206ff90b05e748c64d29d82f7a16
ms.openlocfilehash: 851dc47842563a7cccb41f78112d63b7e59f2e4f
ms.contentlocale: zh-cn
ms.lasthandoff: 04/12/2017

---
# <a name="39catch39-block-never-reached-because-39ltname1gt39-inherits-from-39ltname2gt39"></a><span data-ttu-id="82ddc-102">永远不会达到的值，因为 catch 块&lt;name1&gt;继承&lt;name2&gt;</span><span class="sxs-lookup"><span data-stu-id="82ddc-102">&#39;Catch&#39; block never reached, because &#39;&lt;name1&gt;&#39; inherits from &#39;&lt;name2&gt;&#39;</span></span>
<span data-ttu-id="82ddc-103">从未到达代码中的 `Catch` 块。</span><span class="sxs-lookup"><span data-stu-id="82ddc-103">A `Catch` block within the code is never reached.</span></span>  
  
 <span data-ttu-id="82ddc-104">默认情况下，此消息是一个警告。</span><span class="sxs-lookup"><span data-stu-id="82ddc-104">By default, this message is a warning.</span></span> <span data-ttu-id="82ddc-105">有关隐藏警告或将警告视为错误的详细信息，请参阅[在 Visual Basic 中配置警告](https://docs.microsoft.com/visualstudio/ide/configuring-warnings-in-visual-basic)。</span><span class="sxs-lookup"><span data-stu-id="82ddc-105">For more information on hiding warnings or treating warnings as errors, see [Configuring Warnings in Visual Basic](https://docs.microsoft.com/visualstudio/ide/configuring-warnings-in-visual-basic).</span></span>  
  
 <span data-ttu-id="82ddc-106">**错误 ID：** BC42029</span><span class="sxs-lookup"><span data-stu-id="82ddc-106">**Error ID:** BC42029</span></span>  
  
## <a name="to-correct-this-error"></a><span data-ttu-id="82ddc-107">更正此错误</span><span class="sxs-lookup"><span data-stu-id="82ddc-107">To correct this error</span></span>  
  
-   <span data-ttu-id="82ddc-108">删除无关的 `Catch` 块。</span><span class="sxs-lookup"><span data-stu-id="82ddc-108">Remove the extraneous `Catch` block.</span></span>  
  
## <a name="see-also"></a><span data-ttu-id="82ddc-109">另请参阅</span><span class="sxs-lookup"><span data-stu-id="82ddc-109">See Also</span></span>  
 <span data-ttu-id="82ddc-110">[如何︰ 对代码进行尝试测试...在 Visual Basic 中 catch 块](http://msdn.microsoft.com/en-us/8368e205-ed73-4185-a247-af84fb4fafa9) </span><span class="sxs-lookup"><span data-stu-id="82ddc-110">[How to: Test Code with a Try…Catch Block in Visual Basic](http://msdn.microsoft.com/en-us/8368e205-ed73-4185-a247-af84fb4fafa9) </span></span>  
<span data-ttu-id="82ddc-111"> [Try...Catch...Finally 语句](../../visual-basic/language-reference/statements/try-catch-finally-statement.md)</span><span class="sxs-lookup"><span data-stu-id="82ddc-111"> [Try...Catch...Finally Statement](../../visual-basic/language-reference/statements/try-catch-finally-statement.md)</span></span>
